docker-compose.override.yml

```diff
@@ -4,5 +4,7 @@ services:
 ###> doctrine/doctrine-bundle ###
   database:
     ports:
-      - "5432"
+      - target: 5432
+        published: ${POSTGRES_PORT_OUTSIDE:-5432}
+        protocol: tcp
```

Comment:
I was too late in prepping for travel to see this in time! But, since it's here, can you explain these last 2 lines? Btw, these lines are directly from the DoctrineBundle recipe: https://github.com/symfony/recipes/blob/main/doctrine/doctrine-bundle/2.4/manifest.json#L41-L44